app/MMB/src/scenes/passenger/Passenger.js

```diff
@@ -48,40 +49,43 @@ export class Passenger extends React.Component<Props> {
     );
 
     return (
-      <SimpleCard style={styles.card}>
-        <Text style={styles.name}>
-          <Translation passThrough={`${title}. ${fullName}`} />
-        </Text>
-        <Row>
-          <Column
-            title={<Translation id="mmb.passenger.nationality.label" />}
-            value={<Translation passThrough={nationality.toUpperCase()} />}
-          />
-          <Column
-            title={<Translation id="mmb.passenger.birthday" />}
-            value={
-              <Translation
-                passThrough={DateFormatter(
-                  new Date(birthday),
-                ).formatToBirthday()}
-              />
-            }
-          />
-        </Row>
-        <View style={styles.separator}>
-          <SeparatorFullWidth />
-        </View>
-        <Row>
-          <Column
-            title={<Translation id="mmb.passenger.id" />}
-            value={<Translation passThrough={idNumber} />}
-          />
-          <Column
-            title={<Translation id="mmb.passenger.insurance" />}
-            value={<Translation passThrough={insuranceType} />}
-          />
-        </Row>
-      </SimpleCard>
+      <React.Fragment>
+        <SimpleCard style={styles.card}>
```

Comment:
Did you try it on Android? `SimpleCard` component is with radiuses so I cannot imagine this works well together... 🙂